core/src/main/java/org/apache/accumulo/core/client/ClientConfiguration.java

```diff
@@ -41,7 +41,22 @@
 
 /**
  * Contains a list of property keys recognized by the Accumulo client and convenience methods for setting them.
- *
+ * <p>
+ * When loading a configuration file from the system, Accumulo uses the <em>ACCUMULO_CLIENT_CONF_PATH</em> environment variable,
```

Comment:
Does the behavior described here only happen when calling `loadDefault()`?  Or do all of the constuctors do this search?

Reply:
Not all the constructors do the search. The described behavior occurs when load configurations use a default load path. Previously the path order was described only in the method. I moved it up to class level based upon the suggestion given by @madrob in the ticket comment.  It can be moved back into the method call only if you think that would be more appropriate.

Reply:
>  It can be moved back into the method call only if you think that would be more appropriate.

Personally I would put the docs on the loadDefault method and then any constuctors that call load default I would state in the constuctors javadoc that it follows the search behavior described in loadDefatult using s javadoc link.  

If the docs are at the class level, I think the class level docs would need to detail which constuctors and methods do and/or do not follow the search behavior.

Reply:
Sounds good. I'll update to make the adjustments.  So if I understand you correctly I will only need 2 PR's.  One for the 1.7 change and another for the 1.8 change and then the 1.8 change will be merged forward to 2.0? Thanks.

Reply:
>  One for the 1.7 change and another for the 1.8 change and then the 1.8 change will be merged forward to 2.0?

So you want the changes in 1.8 to be slightly different?  If so it would be nice to end up with the following on 1.8 branch.

 * Your commit with 1.8 changes
 * Merge commit with your changes from 1.7 
 * Your 1.7 commit

Lets go for the easiest way to get to those changes.  Lets get the changes in 1.7 squared away and then we can get the 1.8 changes figured out.